taggui/auto_captioning/captioning_thread.py

```diff
@@ -407,7 +407,7 @@ def run(self):
             try:
                 model_inputs = self.get_model_inputs(image, prompt, model_type,
                                                      device, model, processor)
-            except UnidentifiedImageError:
+            except Exception as exception: # UnidentifiedImageError
```

Comment:
What is the reason for this change?

I think it's better to catch the more specific exception, because other types of exceptions are not expected to occur here.

Reply:
the output is independent of any specific infos, the treatment is to skip the image, so i thought it's applicable for any exception

Reply:
I understand, but the error message actually specifies the reason for skipping the image as "its file format is not supported or it is a corrupted image", which is what causes the `UnidentifiedImageError`. So only that exception is considered here, and the occurence of any other exception should be investigated instead of potentially being ignored.